src/charm.py

```diff
@@ -155,7 +155,8 @@ def _max_body_size(self):
     @property
     def _owasp_modsecurity_crs(self):
         """Return a boolean indicating whether OWASP ModSecurity CRS is enabled."""
-        return self._get_config_or_relation_data("owasp-modsecurity-crs", False)
+        value = self._get_config_or_relation_data("owasp-modsecurity-crs", False)
+        return str(value).lower() == "true"
```

Comment:
Is `lower()` needed ? The latest yaml spec has only "true" and "false" as boolean scalars

Reply:
The ingress library will pass "True" and "False" in relation data, so it's needed here.

Reply:
Add to BOOLEAN_CONFIG_FIELDS wouldn't' fix that?
Or this should be done for rewrite-enabled as well?

Reply:
It's already be done for `rewrite-enabled`, but you are right about adding this to `BOOLEAN_CONFIG_FIELDS`

Reply:
Hi Amanda, see the discussion down below, I think I will leave this problem to the future.